### ISSUE TITLE
[Feature Request] Add an exclusion for timed backups

### ISSUE THREAD

**Issue by Alfie51m** [MaxMaeder/DriveBackupV2#204]
### What feature do you want to see added?

For example I have a scheduled daily restart every 24 hours, which causes a backup to start around the time my server restarts. Would be nice if I could have the backup delay set normally, but add an exclusion to prevent it happening in a period of time

### Are there any alternatives?

Manually setting it with a schedule, however takes a lot more time to setup

**Comment by lrmtheboss** [comment]
Basically another backup schedule but as  exclusions for delay based backups? Wouldn't it just be easier to use backup schedules?

**Comment by Alfie51m** [comment]
Not really, instead of lets say having to make a backup schedule 23 times for each hour as per your documentation you could instead leave the delay at 60 and exclude 1am to prevent backups during that time
